### DISCUSSION TITLE
Add entitlements for services without creating them

### DISCUSSION THREAD

**Discussion by AnatolManikalo** [SAP-archive/btp-setup-automator#397] [Q&A] [answered]
Hello,

Is there a possibility to add entitlements for different services without creating them actually during the setup?
If not, maybe it makes sense to enhance usecases with "category": "entitlement" so to enable the following user story:

As a subaccount creator, I want to provide a list of services and subscriptions so that corresponding quotas are added to the subaccount. Developers then can use this quota for their needs without requesting it from the subaccount/GA admins.

I understand this can be achieved by "executeAfterAccountSetup", but specifying them as JSON objects looks more convenient to use.

Thanks,
Anatol

**Comment by lechnerc77** [comment, marked as answer]
This can be achieved via the `entitleonly` parameter you have available for your services in the `usecase.json`

See also Documentation: <https://github.com/SAP-samples/btp-setup-automator/blob/main/docs/generated/btpsa-usecase-properties-services-items.md#entitleonly>
